django_app/report_a_suspected_breach/views.py

```diff
@@ -595,3 +596,14 @@ def get(self, *args: object, file_name, **kwargs: object) -> HttpResponse:
             return redirect(file_url)
 
         raise Http404()
+
+
+class DeleteEndUserView(View):
+    def post(self, *args: object, **kwargs: object) -> HttpResponse:
+        if end_user_uuid := self.request.GET.get("end_user_uuid"):
```

Comment:
related to the comment above, this would change to:

```suggestion
        if end_user_uuid := self.request.POST.get("end_user_uuid"):
```